evalml/pipelines/components/ensemble/sklearn_stacked_ensemble_base.py

```diff
@@ -10,7 +11,7 @@
 class SklearnStackedEnsembleBase(Estimator):
     """Stacked Ensemble Base Class.
 
-    Arguments:
+    Args:
```

Comment:
Is there supposed to be a "Raises" part of the docstring here?

Reply:
Yeah, though I think these linting packages aren't smart enough to pick up that a class docstring should be associated with the __init__ docstring 😅